client/app/packs/stores-registration.js

```
@@ -1,4 +1,7 @@
-import ReactOnRails from 'react-on-rails';
+// 'use client' keeps this pack and its store imports out of the RSC bundle.
+'use client';
```

ESLint

[error] Expected newline before "use client" directive.

(lines-around-directive)